RemoteWellnessSupportApp/Common/Models/PhysicalCondition.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  PhysicalCondition.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/02/18.
+//
+
+import Foundation
+import SwiftData
+
+@Model
+class PhysicalCondition {
+    @Attribute(.unique) var id: String = UUID().uuidString
+    var memo: String
+    var rating: Int
+    var entryDate: Date
+    var createdAt: Date
+    var updatedAt: Date
+
+    init(id: String = UUID().uuidString,
+         memo: String, rating: Int, entryDate: Date, createdAt: Date, updatedAt: Date) {
```

Comment:
[nits]
タイムスタンプは、生成時の時刻を初期値に設定することが大半なので、デフォルト値を指定しておくと使いやすいと思います。

```suggestion
         memo: String, rating: Int, entryDate: Date, createdAt: Date = Date(), updatedAt: Date = Date()) {
```

Reply:
こちらにて対応しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/pull/6/commits/d6ad39641395818ce9c31ff1a694af5d7c014743